src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="starsArr">
+        <div class="stars--0">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="stars--1">
+          <div class="stars--N"></div>
```

Comment:
it should be just starts__star, colored stars should be controlled by parent class stars--N, where N is a number of colored stars 
![Screenshot 2023-05-03 at 17 15 03](https://user-images.githubusercontent.com/6526149/235942958-b3ef448a-f0a0-4fe8-b1ec-d8b03c065ee8.png)